pages/features/internationalization.mdx

```diff
@@ -0,0 +1,44 @@
+import { ImageTabs } from '../../components/ImageTabs';
+
+Start UI already implements internationalization, which means that it is designed to support multiple languages.
+It currently supports French, English, Arabic and Swahili, and is set up to display text in these languages in a way that is appropriate
+for the language's writing system. For example, if you are using Start UI in Arabic, the text will be displayed right-to-left, as is standard for Arabic.
+Similarly, if you are using Start UI in Swahili, the text will be displayed left-to-right, as is standard for languages that use the Latin alphabet.
+This allows users to access the user interface in their preferred language, making it more accessible and user-friendly for people around the world.
+The supported locales are available in [DayJS' documentation](https://day.js.org/docs/en/i18n/i18n).
+The default language is English and the font size of each implemented language can be adjusted manually (see **Guides / Adding a new language**).
```

Comment:
mention that it is to be developed. The best way to do it would be to create the page as WIP and link to this page